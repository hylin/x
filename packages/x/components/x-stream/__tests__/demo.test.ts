import { baseTest } from '../../../tests/shared/demoTest';

baseTest(false, 'x-stream');
