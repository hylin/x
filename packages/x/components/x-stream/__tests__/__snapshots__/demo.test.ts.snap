// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/x-stream/demo/custom-protocol.tsx correctly 1`] = `
<div
  class="ant-splitter ant-splitter-horizontal"
>
  <div
    class="ant-splitter-panel"
    style="flex-basis:auto;flex-grow:1"
  >
    <button
      class="ant-btn ant-btn-primary ant-btn-color-primary ant-btn-variant-solid"
      style="margin-bottom:16px"
      type="button"
    >
      <span>
        Mock Custom Protocol - SIP
      </span>
    </button>
  </div>
  <div
    aria-valuemax="0"
    aria-valuemin="0"
    aria-valuenow="50"
    class="ant-splitter-bar"
    role="separator"
  >
    <div
      class="ant-splitter-bar-dragger"
    />
  </div>
  <div
    class="ant-splitter-panel"
    style="margin-left:16px;flex-basis:auto;flex-grow:1"
  >
    <div
      class="ant-thought-chain ant-thought-chain-middle"
    />
  </div>
</div>
`;

exports[`renders components/x-stream/demo/default-protocol.tsx correctly 1`] = `
<div
  class="ant-splitter ant-splitter-horizontal"
>
  <div
    class="ant-splitter-panel"
    style="flex-basis:auto;flex-grow:1"
  >
    <button
      class="ant-btn ant-btn-primary ant-btn-color-primary ant-btn-variant-solid"
      style="margin-bottom:16px"
      type="button"
    >
      <span>
        Mock Default Protocol - SSE
      </span>
    </button>
  </div>
  <div
    aria-valuemax="0"
    aria-valuemin="0"
    aria-valuenow="50"
    class="ant-splitter-bar"
    role="separator"
  >
    <div
      class="ant-splitter-bar-dragger"
    />
  </div>
  <div
    class="ant-splitter-panel"
    style="margin-left:16px;flex-basis:auto;flex-grow:1"
  >
    <div
      class="ant-thought-chain ant-thought-chain-middle"
    />
  </div>
</div>
`;
